go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/adobe-apiplatform/api-gateway-config-supervisor
+
+go 1.20
```

Comment:
API Gateway was updated to 1.23 https://github.com/adobe-apiplatform/apigateway/pull/83/files#diff-2d91ce62c68cf73dc917ebfe74dcbc10fbe2d64cc2244ed63b6e321c72d1d0e7R3, I'm not sure if they have to be on the same version.